lib/open_api_spex/cast/array.ex

```diff
@@ -14,6 +14,14 @@ defmodule OpenApiSpex.Cast.Array do
     end
   end
 
+  def cast(%{value: %{"_json" => items}} = ctx) when is_list(items) do
```

Comment:
I think the best place to handle this special case is in cast_request_body: https://github.com/open-api-spex/open_api_spex/blob/master/lib/open_api_spex/operation2.ex#L40

That way we can be sure this is only applying to the top level request body schema.